docs/gestorTareas.md

```diff
@@ -0,0 +1,7 @@
+# Gestor de Tareas (INVOKE)
+
+Como gestor de tareas tenía varias posibilidades para el lenguaje que estoy utilizando, invoke, pypyr o make. Los criterios que voy a utilizar para elegir uno u otro serán, documentación, sintaxis sencilla y usabilidad. Siguiendo esas pautas Invoke es el gestor de tareas que voy a utilizar, ya que cuenta con una buena documentación, la sintaxis a la hora de declarar y gestionar las tareas me parece más cómoda que si utilizase make y por último he visto que es altamente utilizado por la comunidad de Python a la hora de desarrollar sus proyectos.
+
+# Gestor de Dependencias (POETRY)
+
+Como gestor de dependencias también tenía varias opciones poetry, pipenv o hatch. En esta ocasión he elegido poetry, ya que su sintaxis es muy sencilla, mucho más que las otras dos opciones. Hatch cuenta con muchas más funcionalidades que poetry, pero a costa de una sintaxis mucho más compleja, además estas opciones no son necesarias en este proyecto. Por último destacar que en proyectos python poetry tiene muchas más menciones y estrellas que los demás.
```

Comment:
Ya lo explicamos en clase múltiples veces.
1. Se enuncian los criterios objetivos para elegir una herramienta u otra
2. Se dice qué herramientas hay que cumplan esos criterios.
3. En función del cumplimiento, se elige o no una herramienta u otra.

En este caso concreto, la sintaxis de hatch es *literalmente* la misma que la de poetry, TOML.

Como hemos explicado múltiples veces en clase, lo que se persigue con los diferentes objetivos es que aprendáis a tomar decisiones informadas en un proyecto. No se trata de que elijáis una cosa y luego busquéis qué poner para pasar el objetivo.

Reply:
De acuerdo